xopt/vocs.py

```diff
@@ -220,8 +223,22 @@ def get_variable_data(
     if not isinstance(data, pd.DataFrame):
         data = pd.DataFrame(data)
 
+    def custom_sort_key(s):
```

Comment:
so actually, the vocs standard is to list things according to the variable names in the dict, not sort them alpha-numerically, so we might want to revert this change to be consistent with the standard